app/src/components/IndicatorData.vue

```diff
@@ -56,15 +59,17 @@
       :width="null"
       :height="null"
       :chart-data='datacollection'
-      :options='chartOptions()'></line-chart>
+      :options='chartOptions()'
+    />
+
     <v-btn
       ref="zoomResetButton"
-      style="position: absolute; right: 40px; top: 13px;display: none;"
+      style="display: none; position: absolute; right: 40px; top: 13px;"
       elevation="2"
-      x-small
-      @click="resetLCZoom"
-    >
-      Reset
+      small
+      @click="resetLCZoom">
+
+      Reset Zoom
```

Comment:
line 72 should be the only difference in this file I think :-) please try to revert the other changes that are not needed

Reply:
Sure thing - but please let's fix the indentation issues within this file in a refactoring session in the figure.

Reply:
Lines 20 and 67